src/Hyperion/Main.hs

```diff
@@ -78,7 +81,8 @@ options = do
          (Options.strOption
             (Options.long "output" <>
              Options.short 'o' <>
-             Options.metavar "FILENAME"))
+             Options.help "Outputs the benchark to the directory" <>
```

Comment:
That's not true, and has been discussed before. I'd rephrase that as "I don't know how to tell whether the user gave us a directory or a file". You can use `doesDirectoryExist` from the `directory` package for this. If the path does not exist, then assume it's a file. If it does exist, and is a directory, then it's a directory. Optional refinement: use `hasTrailingPathSeparator` in `filepath` to create a directory if it doesn't exist, because in that case it's clear that the user does want a directory.

`rsync` and other standard unix tools use a similar convention.